### PR TITLE
feat(demo): update landing page

### PR DESCRIPTION
**Pull request by ghost**
Closes to #4970

### CODE REVIEW

**Review by valorkin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -172,8 +172,8 @@ Please read our [contribution guidelines](https://github.com/valor-software/ngx-
 Please read central `ngx` modules [readme](https://github.com/valor-software/ng2-plans) for details, plans and approach
 
 ### Credits
-Crossbrowser testing sponsored by [Browser Stack](https://www.browserstack.com)
-[<img src="https://camo.githubusercontent.com/a7b268f2785656ab3ca7b1cbb1633ee5affceb8f/68747470733a2f2f64677a6f7139623561736a67312e636c6f756466726f6e742e6e65742f70726f64756374696f6e2f696d616765732f6c61796f75742f6c6f676f2d6865616465722e706e67" alt="Browser Stack" height="31px" style="background: cornflowerblue;">](https://www.browserstack.com)
+Crossbrowser testing sponsored by [Saucelab](https://saucelabs.com/)
+[<img src="https://saucelabs.com/content/images/circle-logo@2x.png" alt="Browser Stack" width="31" height="31">](https://saucelabs.com/)
```
Comment (ludmilanesvitiy) [inline README.md:176]: by [Saucelabs]
alt="Saucelabs"

File: demo/src/app/common/getting-started/getting-started.component.html [hunk 2]
```diff
@@ -166,8 +167,10 @@ <h3>If you need more modules check <a href="https://github.com/valor-software/ng
       <h3>Credits</h3>
 
       <p>Crossbrowser testing sponsored by
-        <a href="https://www.browserstack.com">Browser Stack</a>
-        <a href="https://www.browserstack.com"><img src="https://camo.githubusercontent.com/a7b268f2785656ab3ca7b1cbb1633ee5affceb8f/68747470733a2f2f64677a6f7139623561736a67312e636c6f756466726f6e742e6e65742f70726f64756374696f6e2f696d616765732f6c61796f75742f6c6f676f2d6865616465722e706e67" alt="Browser Stack" height="31px"></a>
+        <a href="https://saucelabs.com/" class="pr-1">Saucelab</a>
```
Comment (ludmilanesvitiy) [inline demo/src/app/common/getting-started/getting-started.component.html:170]: Saucelabs

File: demo/src/app/common/getting-started/getting-started.component.html [hunk 3]
```diff
@@ -166,8 +167,10 @@ <h3>If you need more modules check <a href="https://github.com/valor-software/ng
...
+        <a href="https://saucelabs.com/">
+          <img src="https://saucelabs.com/content/images/circle-logo@2x.png" alt="Browser Stack" width="31" height="31">
```
Comment (ludmilanesvitiy) [inline demo/src/app/common/getting-started/getting-started.component.html:172]: alt="Saucelabs"

**Review by ludmilanesvitiy — COMMENTED** [review]
Also need to change link "There's no need to open an issue here if you want to ask general question, use [StackOverflow](https://stackoverflow.com/questions/tagged/ngx-bootstrap) or [Slack](https://ngx-slack.herokuapp.com/) instead" - in ISSUE_TEMPLATE.md

Also need to fix Cypress tests

Also need to fix links in CONTRIBUTING.md file

**Pull request merged**